contrib/imgprune/entrypoint.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+set -e
+
+source /usr/local/bin/lib_entrypoint.sh
+
+req_env_var GCPJSON GCPNAME GCPPROJECT IMGNAMES
+
+gcloud_init
+
+# Different purpose from quay.io/libpod/imgts
+EXCLUDE="$IMGNAMES $IMAGE_BUILDER_CACHE_IMAGE_NAME"
+# For safety's sake + limit nr background processes
+PRUNE_LIMIT=10
+THEFUTURE=$(date --date='+1 hour' +%s)
+THRESHOLD=$(date --date='90 days ago' +%s)
+FORMAT='value[quote](name,selfLink,creationTimestamp,labels)'
+PROJRE="/v1/projects/$GCPPROJECT/global/"
+BASE_IMAGE_RE='cloud-base'
+FILTER="selfLink~$PROJRE AND creationTimestamp<$(date --date='30 days ago' --iso-8601=date) AND NOT name=($EXCLUDE)"
```

Comment:
May I safely assume that this `name=(...)` filtering works fine with a space-separated list of multiple patterns? Can a future maintenance programmer also assume that? This might be a great place for a reassuring comment.

Reply:
Actually, this filtering/formatting is complex enough, probably dropping actual documentation in here would be a good idea also.   Okay got it.